requirements.txt

```diff
@@ -0,0 +1,33 @@
+alembic==0.9.1
+altgraph==0.13
+aniso8601==1.2.0
+appdirs==1.4.3
+click==6.7
+colorama==0.3.7
```

Comment:
Some requirements here are not unnecessary. Why do you need `colorama` and `click` for example?